libraries/localization-utilities/src/parsers/test/parseLocJson.test.ts

```diff
@@ -1,23 +1,36 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import { IgnoreStringFunction } from '../../interfaces';
 import { parseLocJson } from '../parseLocJson';
+import { serializeLocFile } from './serializeLocFile';
 
 describe(parseLocJson.name, () => {
   it('parses a valid file', () => {
     const content: string = JSON.stringify({
       foo: {
         value: 'Foo',
         comment: 'A string'
+      },
+      bar: {
+        value: 'Bar',
+        comment: 'Another string'
       }
     });
 
     expect(
-      parseLocJson({
-        content,
-        filePath: 'test.loc.json'
-      })
-    ).toMatchSnapshot();
+      serializeLocFile(
+        parseLocJson({
+          content,
+          filePath: 'test.loc.json'
+        })
+      )
+    ).toMatchInlineSnapshot(`
+      "key | value | comment
```

Comment:
Can we stick with the external snapshots? I'm more comfortable with keeping data and logic separate and, importantly, not having build steps modify code files.

Reply:
Also, sorry to say, but I find this table harder to read than the JSON, since it makes whitespace (which is a rather significant element of the parsers) hard to compare. It is completely valid for strings to contain tabs, newlines, etc.

Reply:
Fair enough.